sale_request/wizards/create_sale_order_wizard_view.xml

```diff
@@ -6,7 +6,7 @@
         <field name="arch" type="xml">
             <form>
                 <div class="alert alert-danger" role="alert" style="margin-bottom:0px;" attrs="{'invisible': [('has_lines', '=', True),('on_time', '=', True)]}">
-                    <bold>Sorry!</bold>, there is no master order for this product. You want to create requisition without order?
+                    <bold>Sorry!</bold>, Do you want to confirm out of schedule?
```

Comment:
This change is wrong, we still need the original label... There are two use cases here

1. When the product you try to sell doesn't have a master sale order. This must show the original label.
2. When it's after 7 pm, in this case it doesn't matter if the product has a master sale order, the person that links them is not working, so you must show the new label.